packages/crawler/src/page-processors/page-processor-base.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import Apify from 'apify';
+import { Logger } from 'logger';
+import { AccessibilityScanOperation } from '../page-operations/accessibility-scan-operation';
+import { ScanData } from '../scan-data';
+import { LocalBlobStore } from '../storage/local-blob-store';
+import { LocalDataStore } from '../storage/local-data-store';
+import { BlobStore, DataStore, scanResultStorageName } from '../storage/store-types';
+import { PageProcessorHelper } from './page-processor-helper';
+
+export interface PageProcessorOptions {
+    baseUrl: string;
+    requestQueue: Apify.RequestQueue;
+    logger: Logger;
```

Comment:
`Logger` is not logically a part of page processor options. We do not want to have `PageProcessorOptions` as bucket for everything